styleguide/_themes/derek/scss/globalElements/basement.scss

```diff
@@ -16,7 +16,7 @@
 
 // Basement Classes
 .basement {
-  padding: 10px 15px;
+  padding: 10px $grid-gutter-width;
```

Comment:
`$grid-gutter-width` is 30px. Is this what we want, as the previous value was 15px.
https://github.com/rackerlabs/zoolander/blob/59e620b11f6807d622b69f17d5c8f8a88ddf4d25/styleguide/_themes/global/scss/bootstrap_variables.scss#L352